client/src/components/UserInfo/index.tsx

```diff
@@ -1,12 +1,15 @@
 import { NavLink } from 'react-router-dom';
+import Avatar from '@mui/material/Avatar';
 import User from '../../Interfaces/post/IUser';
 
-const UserPostInfo = ({ user, style }:{user:User, style:{
-  dimensions: number, alignItems: string
-}}) => (
-  <NavLink style={{ alignItems: style.alignItems }} to={`/users/${user.id}`} className="user-info-post">
-    <img style={{ width: style.dimensions, height: style.dimensions }} src={user.avatar} alt="" />
-    <h4>{user.name}</h4>
+const UserPostInfo = ({ user }:{user:User}) => (
+  <NavLink to={`/users/${user.id}`} className="user-info-post">
+    <div style={{ display: 'flex' }}>
+      <Avatar alt={user.name} src={user.avatar} />
+      <span style={{ alignSelf: 'center', paddingLeft: '10px' }}>
```

Comment:
Use mui's `Typography`
`<Typography variant='caption'>`
replace caption with any of the relevant options
This is good in case you change the font size, Typography refers to the MUI theme when its used